haechan45th/src/services/userService.js

```diff
@@ -16,7 +20,13 @@ const signUp = async (name, email, password, profileImage) => {
         emailValidationCheck(email);
         pwValidationCheck(password);
 
-        const createUser = await userDao.createUser(name, email, password, profileImage);
+        const makeHash = async (password, saltRounds) => {
+            return await bcrypt.hash(password, saltRounds);
+        }
+
+        const hashedPassword = await makeHash(password, saltRounds);
```

Comment:
```suggestion
        const saltRounds = 12;
        const hashedPassword = await bcrypt.hash(password, saltRounds);
```
위에서 남겨드렸던 전역 변수 대신 지역 변수로서 선언하여 사용하면 됩니다. 
또한 makeHash 함수는 내부적으로 어떠한 로직도 갖고 있지 않으므로 곧바로 hashedPassword에서 작성될 수 있도록 해주세요!